libraries/AP_RangeFinder/AP_RangeFinder_MAVLink.cpp

```diff
@@ -69,7 +69,7 @@ void AP_RangeFinder_MAVLink::update(void)
         set_status(RangeFinder::RangeFinder_NoData);
         state.distance_cm = 0;
     } else {
-        set_status(RangeFinder::RangeFinder_Good);
         state.distance_cm = distance_cm;
+        update_status();
```

Comment:
Was this changed for some specific reason? I think it was done this way so that a MAVLink rangefinder didn't need any min/max limit configured.